src/components/KonnectorFolder.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { translate } from '../plugins/i18n'
+import styles from '../styles/konnectorFolder'
+
+import DescriptionContent from './DescriptionContent'
+
+export const KonnectorFolder = ({ t, account, driveUrl, connector }) => {
```

Comment:
In my opinion `KonnectorFolder` could be a stateful component and fetch `driveUrl` by himself in `componentDidMount`.